code/core/src/manager/components/preview/Toolbar.tsx

```diff
@@ -172,12 +181,14 @@ export const ToolbarComp = React.memo<ToolData>(function ToolbarComp({
 
 export const Tools = React.memo<{ list: Addon_BaseType[] }>(function Tools({ list }) {
   return (
-    <>
+    <ToolList>
       {list.filter(Boolean).map(({ render: Render, id, ...t }, index) => (
         // @ts-expect-error (Converted from ts-ignore)
-        <Render key={id || t.key || `f-${index}`} />
+        <ToolListItem key={id || t.key || `f-${index}`}>
+          <Render />
+        </ToolListItem>
```

Comment:
Did you check if everything looks after the change? Do we have chromatic visuals for those components?

Reply:
How much does mean 'everything'? As far as I checked, generally no problem was found in the view.
It looks like there is no story with Toolbar.